pandas/core/frame.py

```diff
@@ -8811,18 +8811,18 @@ def append(
 
         Examples
         --------
-        >>> df = pd.DataFrame([[1, 2], [3, 4]], columns=list('AB'))
+        >>> df = pd.DataFrame([[1, 2], [3, 4]], columns=list('AB'), index=[0, 2])
         >>> df
            A  B
         0  1  2
-        1  3  4
-        >>> df2 = pd.DataFrame([[5, 6], [7, 8]], columns=list('AB'))
+        2  3  4
+        >>> df2 = pd.DataFrame([[5, 6], [7, 8]], columns=list('AB'), index=[0, 2])
```

Comment:
Could you use non matching indexes?

Reply:
Sorry, this was not clear enough. Could we do something like ``[0, 4]`` to show that indexes get duplicated when they have the same element and the second element is non-matching to show the other case? Should have made the more clear before, sorry.

Reply:
Sure, there you go :)